Modules/Client/Entities/Client.php

```diff
@@ -239,4 +249,31 @@ public function getClientMonthEndDateAttribute()
             return today(config('constants.timezone.indian'))->addMonthsNoOverflow()->startOfMonth()->addDays($billingDate - 2);
         }
     }
+
+    public function getCcEmailsAttribute()
+    {
+        $ccEmails = null;
+        if (optional($this->secondary_contact)->first() != null) {
+            $ccEmails = config('invoice.mail.send-invoice.email');
+            foreach ($this->secondary_contact as $ccEmail) {
+                $ccEmails .= ",$ccEmail->email";
+            }
+        }
+
+        return $ccEmails;
+    }
+
+    public function getBccEmailsAttribute()
+    {
+        $bccEmails = null;
+        if (optional($this->tertiary_contact)->first() != null) {
+            $bccEmails = '';
+            foreach ($this->tertiary_contact as $bccEmail) {
+                $bccEmails .= ",$bccEmail->email";
```

Comment:
Not sure how this is working correct, as you have passed variable inside a string.
```suggestion
                $bccEmails .= $tertiaryContact->email . ",";
```